main/run.py

```diff
@@ -26,19 +27,93 @@
     URBAN_BRIGHTNESS_THRESHOLD,
     URBAN_NEARBY_RADIUS,
     MIN_NEARBY_RURAL_STATIONS,
-    START_DATE,
-    END_DATE,
-    BASELINE_START_DATE,
-    BASELINE_END_DATE,
     SST_CUTOFF_TEMP,
+    URBAN_ADJUSTMENT_OPTION,
 )
 
 
+def parse_arguments():
+    """
+    Parse command-line arguments for executing the GISTEMP algorithm.
+
+    Returns:
+        argparse.Namespace: Parsed command-line arguments.
+    """
+    parser = argparse.ArgumentParser(
+        description="Execute steps of the GISTEMP algorithm."
```

Comment:
How detailed do we want to make the argument help / error messages? Even if explained in the read me, might be helpful to have super detailed explanation of each of the arguments and how to use. Think something like nco. If you mess up the command or use -help you can pretty much solve any issue with just the error message

Reply:
Going to add some error messages for the next PR, good call!